sync_pr_to_gitlab/github_pr_to_internal_pr.py

```diff
@@ -40,7 +41,7 @@ def pr_check_approver(pr_creator, pr_comments_url, pr_approve_labeller):
 
     for comment in reversed(r_data):
         comment_body = comment['body']
-        if bool(re.match('sha=', comment_body, re.I)) and comment['user']['login'] == pr_approve_labeller != pr_creator:
```

Comment:
Could we keep this check for `espressif/esp-idf`?

The check prevents an ESP-IDF developer from creating and syncing a PR to GitLab single-handedly, which should be a very rare occurrence.

Reply:
I see. I extended the condition to not allow `pr_approve_labeller == pr_creator` in esp-idf.